### PR TITLE
Add extarnal id venn mapping for region

### PR DESCRIPTION
**Pull request by VinoLora**

### CODE REVIEW

File: src/graphql/handler/handler.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { whitelistSchema } from '../generated';
 
 sq.init(models);
 
-const LAMBDA_OPTIONS = { promisifyHandler: true, useDbTransactions: true };
+const LAMBDA_OPTIONS = { promisifyHandler: true, useDbTransactions: true, sequelizeModels: models };
```
Comment (SegevHaviv) [inline src/graphql/handler/handler.ts:12]: If we add this, we can get rid of the sq.init above I think

File: sequelize/seeders/V20210119105642__seed_venn_hood_region_mapping.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const config = require('@venncity/nested-config')(__dirname);
+const { createEntityWithValidation } = require('../seedHelpers');
+
+const KANSAS_CITY_MACFORCE_ID = config.get('seed.KANSAS_CITY_MACFORCE_ID');
```
Comment (SegevHaviv) [inline sequelize/seeders/V20210119105642__seed_venn_hood_region_mapping.js:4]: Why production config is empty? Wouldn't it seed empty entities?

File: sequelize/seeders/V20210119105642__seed_venn_hood_region_mapping.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+async function createVennMacforceMapping(macforce, venn) {
+  const whereUnique = { macforce: { entityId: macforce.entityId }, venn: { entityId: venn.entityId } };
+  const createInput = { macforce: { connect: { id: macforce.id } }, venn: { connect: { id: venn.id } } };
+  return createEntityWithValidation({ whereUnique, entity: createInput, entityName: 'VennToMacforceMapping' });
+}
+
+module.exports = {
+  async up() {
+    const macforceExternalId = await createVennMacforceExternalId({ entityName: 'Hood', entityId: KANSAS_CITY_HOOD_ID });
```
Comment (SegevHaviv) [inline sequelize/seeders/V20210119105642__seed_venn_hood_region_mapping.js:20]:
```suggestion
    const [macforceExternalId, vennExternalId] = await Promise.all([createVennMacforceExternalId({ entityName: 'Hood', entityId: KANSAS_CITY_HOOD_ID }), createVennMacforceExternalId({ entityName: 'Region__c', entityId: KANSAS_CITY_MACFORCE_ID })];
```

File: .sequelizerc [hunk 4]
```diff
@@ -2,6 +2,7 @@ const path = require('path');
 
 const { initLocalEnv } = require('@venncity/jest-config-server');
 initLocalEnv();
+require('./sequelize/initSequelizeModel');
```
Comment (SegevHaviv) [inline .sequelizerc:5]: This one will be called before the migrations... I saw this in other services as well but we might want to reconsider this solution in the future ( I guess we'll not delay ourselves about it for now )

**Review by SegevHaviv — APPROVED** [review]

File: sequelize/seeders/V20210119105642__seed_venn_hood_region_mapping.js (hunk 2 above)
Comment (VinoLora) [reply sequelize/seeders/V20210119105642__seed_venn_hood_region_mapping.js:4]: I don't know it's id yet.We will add it when we'll have it .

File: sequelize/seeders/V20210119105642__seed_venn_hood_region_mapping.js (hunk 3 above)
Comment (VinoLora) [reply sequelize/seeders/V20210119105642__seed_venn_hood_region_mapping.js:20]: It won't work, I don't know why and I don't have time to dig into it.

File: .sequelizerc (hunk 4 above)
Comment (VinoLora) [reply .sequelizerc:5]: We might.But this is the only way it works now.

**Pull request merged**